tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
-    "module": "none",
+    "module": "ESNext",
     "jsx": "react",
     "target": "es6",
     "moduleResolution": "node",
+    "allowSyntheticDefaultImports": true,
```

Comment:
@blcham According to [this documentation](https://unional.gitbooks.io/typescript/content/pages/06-files-and-projects/compiler-options.html), `allowSyntheticDefaultImports` does not have any implication:

> Allow default imports from modules with no default export. This does not affect code emit, just typechecking. 

However, they still recommend the use of `esModuleInterop` instead:

> Why? Since this flag does not affect code emit, it creates a discrepency between the typings and the actual emitted code. This causes a systematic issue at consumption rendering your package is not usable in certain ecosystem. The only place that make sense to use this flag is at the application level. But even that, you can just use esModuleInterop instead.
> 
>     You can learn more about this in https://github.com/Microsoft/TypeScript/issues/7398

For what it's worth, here is some explanation: https://chat.openai.com/share/3bb73e47-e805-4b43-bc0a-7aa4caf88796

Reply:
Resolving this as the investigation was formulated in https://github.com/kbss-cvut/fta-fmea-ui/issues/151#issuecomment-1911790017.